dtfs-central-api/api-tests/v1/utilisation-reports/get-utilisation-reports.api-test.ts

```diff
@@ -1,61 +1,123 @@
 import { Response } from 'supertest';
-import { MONGO_DB_COLLECTIONS, UtilisationReport } from '@ukef/dtfs2-common';
-import wipeDB from '../../wipeDB';
+import { AzureFileInfo, IsoDateTimeStamp, ReportPeriod, UtilisationReportEntity, UtilisationReportReconciliationStatus } from '@ukef/dtfs2-common';
+import { SqlDbDataSource } from '@ukef/dtfs2-common/sql-db-connection';
+import axios from 'axios';
+import {
+  aNonUploadedMarkedReconciledUtilisationReportEntity,
+  aNotReceivedUtilisationReportEntity,
+  anUploadedUtilisationReportEntity,
+} from '../../mocks/entities/utilisation-report-entity';
 import app from '../../../src/createApp';
-import db from '../../../src/drivers/db-client';
-import { MOCK_UTILISATION_REPORT } from '../../mocks/utilisation-reports/utilisation-reports';
 import createApi from '../../api';
+import { wipeAllUtilisationReports } from '../../test-helpers/wipe-sql-db';
 
 const api = createApi(app);
 
 const getUrl = (bankId: string) => `/v1/bank/${bankId}/utilisation-reports`;
 
+const saveReportToDatabase = async (report: UtilisationReportEntity): Promise<UtilisationReportEntity> =>
+  await SqlDbDataSource.getRepository(UtilisationReportEntity).save(report);
+
+type UtilisationReportResponseBody = {
+  id: number;
+  bankId: string;
+  status: UtilisationReportReconciliationStatus;
+  uploadedByUserId: string;
+  reportPeriod: ReportPeriod;
+  azureFileInfo: AzureFileInfo | null;
+  dateUploaded: IsoDateTimeStamp;
+};
+
 interface CustomResponse extends Response {
   body:
     | {
         errors: {
           msg: string;
         }[];
       }
-    | ({ errors?: undefined } & UtilisationReport);
+    | ({ errors?: undefined } & UtilisationReportResponseBody[]);
 }
 
 describe('GET /v1/bank/:bankId/utilisation-reports', () => {
   beforeAll(async () => {
-    await wipeDB.wipe([MONGO_DB_COLLECTIONS.UTILISATION_REPORTS]);
+    await SqlDbDataSource.initialize();
+    await wipeAllUtilisationReports();
+  });
+
+  afterEach(async () => {
+    await wipeAllUtilisationReports();
   });
 
   it('returns 400 when an invalid bank id is provided', async () => {
     // Act
-    const response: CustomResponse = await api.get(getUrl('invalid-mongo-id'));
+    const response: CustomResponse = await api.get(getUrl('invalid-id'));
```

Comment:
If we introduce the stuff I've commented above, I think the `: CustomResponse` type annotation won't be needed as we will be asserting type with the helper functions which are themselves just expected a `Response` (which I think `api.<fn>` will return by default)